src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -69,6 +69,8 @@ DlgPrefWaveform::DlgPrefWaveform(QWidget* pParent, MixxxMainWindow* pMixxx,
             this, SLOT(slotSetNormalizeOverview(bool)));
     connect(beatGridLinesCheckBox, SIGNAL(toggled(bool)),
             this, SLOT(slotSetGridLines(bool)));
+    connect(dimModeCheckBox, SIGNAL(toggled(bool)),
```

Comment:
beatGridSemiTransparentCheckBox, or a reasonable shorter name

Reply:
Should we have a slider to let the user select their own desired opacity for the lines?

Reply:
I'm for a slider. Just let the opaque value be a float between 0.0-1.0 and a slider for setting it. Default would then be 0.9.